test/ssl_test_ctx.c

```diff
@@ -95,6 +95,7 @@ static const test_enum ssl_test_results[] = {
     {"ServerFail", SSL_TEST_SERVER_FAIL},
     {"ClientFail", SSL_TEST_CLIENT_FAIL},
     {"InternalError", SSL_TEST_INTERNAL_ERROR},
+    {"TimeoutError", SSL_TEST_TIMEOUT_ERROR},
```

Comment:
Added TimeoutError because the NO_RENEGOTIATION alert generated by the receipt of a HelloRequest is never received by the server in the test infrastructure. Instead, the code keeps spinning until it basically times out. If the alert could be delivered to the server, this could be removed.